tests/issues/issue1675/issue1675.icry

```diff
@@ -0,0 +1,5 @@
+:m PrimeEC
+ec_mult`{13} 3 {x = 0, y = 0, z = 1}
+:t ec_mult`{13}
+ec_mult`{13} 3 {x = 0, y = 0, z = 1} : ProjectivePoint 13
+{x = 0, y = 13, z = 1} : ProjectivePoint 13
```

Comment:
We probably don't need line (5), this is just type-checking stuff.

Reply:
Removed in [dd11250](https://github.com/GaloisInc/cryptol/pull/1701/commits/dd112501ca1d3f08350fedf481267d3631832bf4)